ports/esp32/esp32_common.cmake

```diff
@@ -161,7 +161,7 @@ idf_component_register(
 set(MICROPY_TARGET ${COMPONENT_TARGET})
 
 # Define mpy-cross flags, for use with frozen code.
-if(NOT IDF_TARGET STREQUAL "esp32c3")
+if(NOT IDF_TARGET MATCHES "^esp32c[36]$")
```

Comment:
Please don't perform checks for the ESP32 in this way.  The ESP-IDF provides proper CMake variables which can be queried to determine the ISA.  This check is both incorrect because it ignores the fact that forthcoming ESP32 chips are RISC-V based as well as the existing models like the "H" series.